### PR TITLE
Add initial Team Practices description to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jessieay**
* Base on discusssion with team
* Would feedback / sign off from each member of team before merging

- [x] @tadhg-ohiggins 

- [x] @adborden 

- [ ] @anthonygarvan 

- [x] @annalee 

- [x] @porta-antiporta

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,17 +1,91 @@
-## Team Processes
+This document describes how the 18F eRegs team has agreed to work together
```
Comment (adborden) [inline CONTRIBUTING.md]: s/18F eRegs/18F Legal/ (we changed the name :P)

Comment (jessieay) [inline CONTRIBUTING.md]: :+1:

**Comment by adborden** [comment]
Thank you Jesse!

**Comment by annalee** [comment]
:+1:

**Comment by porta-antiporta** [comment]
:tada: @jessieay

File: CONTRIBUTING.md (hunk 1 above)
Comment (jessieay) [inline CONTRIBUTING.md]: updated at aa30b15

**Comment by porta-antiporta** [comment]
@tadhg-ohiggins @anthonygarvan We good to merge this PR?

**Pull request merged**